src/provider/systemdmanager/systemdunit.rs

```diff
@@ -554,6 +575,40 @@ mod test {
             [Install]
             WantedBy=multi-user.target"#}
     )]
+    #[case::with_container_on_session_bus_stoptimeout(
+    BusType::Session,
+    indoc! {r#"
+            apiVersion: v1
+            kind: Pod
+            metadata:
+              name: stackable
+            spec:
+              terminationGracePeriodSeconds: 10
+              containers:
+                - name: test-container.service
+                  command:
+                    - start.sh
+                  securityContext:
+                    windowsOptions:
+                      runAsUserName: container-user
+              securityContext:
+                windowsOptions:
+                  runAsUserName: pod-user"#},
```

Comment:
I would remove all unnecessary stuff to keep the test cases concise and focused.

Reply:
I've stripped it down as much as possible (I think)